### PR TITLE
Updated currentRoute & remove hidden for routechange

### PR DESCRIPTION
**Pull request by gaurav-thoughtspot**
https://thoughtspot.atlassian.net/browse/SCAL-101584

### CODE REVIEW

File: src/embed/liveboard.ts [hunk 1]
```diff
@@ -165,10 +165,7 @@ export class LiveboardEmbed extends V1Embed {
     };
 
     private handleRouteChangeFullHeightLiveboard = (data: MessagePayload) => {
-        if (
-            data.data.canvasState !== 'EMBED' &&
-            data.data.canvasState !== 'pinboard'
-        ) {
+        if (!data.data.currentPath.startsWith('/embed/viz/')) {
```
Comment (ashubham) [inline src/embed/liveboard.ts:168]: Question here. So we do not want to set the height when the route is not a pinboard ? When will this happen ? And the function being named `handleRouteChangeFullHeightLiveboard` feels a bit convoluted.

**Review by ashubham — APPROVED** [review]

**Review by ashubham — CHANGES_REQUESTED** [review]
approved by mistake. Please see comment on PR

File: src/embed/liveboard.ts (hunk 1 above)
Comment (kailash-choudhary-thoughtspot) [inline src/embed/liveboard.ts:168]: here we are setting the default height. If we move away from the embed-pinboard route we set the height of Iframe as the default height provided by user.

Comment (ashubham) [reply src/embed/liveboard.ts:168]: I see. This function seems to handle the case when the route is not "/embed/viz" but is named "handleRouteChangeForFullHeightLiveboard" isnt that convoluted ?

**Review by ashubham — APPROVED** [review]

**Pull request merged**